network-api/networkapi/wagtailpages/utils.py

```diff
@@ -49,12 +50,26 @@ def get_descendants(node, list, authenticated=False, depth=0, max_depth=2):
     helper function to get_menu_pages for performing a depth-first
     discovery pass of all menu-listable children to some root node.
     '''
+
+    def get_overview_label(language_code):
+        localized_overview = {
+            'en': 'Overview',
+            'de': 'Übersicht',
+            'es': 'Síntesis',
+            'fr': 'Vue d’ensemble',
+        }
+
+        overview_label = localized_overview.get(language_code)
+        overview_label = overview_label if overview_label else localized_overview.get('en')
+
+        return overview_label
+
     if (depth <= max_depth):
         title = node.title
         header = getattr(node.specific, 'header', None)
         if header:
             title = header
-        menu_title = title if depth > 0 else 'Overview'
+        menu_title = title if depth > 0 else get_overview_label(translation.get_language())
```

Comment:
This does not seem to follow the standard way of using localized strings in Django (using `gettext` in code and `manage makemessages` to aggregate translation strings). Is there a reason we're not using that?

Reply:
Oooh, I didn’t realize that Overview string was in a Django file!
In that case yes, we should use gettext to localize it. Since the gettext PR hasn’t been merged yet, I’ve got that string wrapped on top of it: https://github.com/mozilla/foundation.mozilla.org/pull/3529/commits/c5abba3569548880dca1ab68450a22e3f1526653#diff-00c96eccdb56461892dffba925859b67

Reply:
I'm not familiar with our localization work. @TheoChevalier does the commit you mentioned above belong to an existing PR? If so, do we still need this PR?

Reply:
Yes, it’s from https://github.com/mozilla/foundation.mozilla.org/pull/3529 so we can close this one.

The TL;DR is that we all thought this required a hack, but it turned out the work in PR 3529 enables us to localize it properly.

Sorry for the extra work you had to do, @mmmavis!

Reply:
No worries at all! It's good for me to slowly familiarize myself with Django and our localization setup!